### ISSUE TITLE
Mission and Vision Isocons

### ISSUE THREAD

**Issue by elizbond** [cfpb/design-manual#254]
Can we revisit this isocon/valucon combo? I see it in a lot of reports and would love to get rid of the creepy hands, as well as make them seem more like a set.

![screen shot 2014-10-02 at 2 11 00 pm](https://cloud.githubusercontent.com/assets/6798337/4495910/49bc062c-4a60-11e4-8daf-ae079c83dfea.png)

![screen shot 2014-10-02 at 2 11 55 pm](https://cloud.githubusercontent.com/assets/6798337/4495920/6a0a9bd2-4a60-11e4-8a52-6827a4810106.png)

**Comment by nataliafitzgerald** [comment]
At this point, we should take a step back and think about the hierarchy for the full set of our: Mission, Vision and Values (words and imagery). There is inconsistency across the board in terms of the words used for our Mission. Can we request feedback from CE on this? @jameshupp would be a good resource. Additionally, we should make sure that our imagery works with the hierarchy. So, the "How we achieve our mission" that appears in GPRA should get minicons or no icons at all rather than being styled in the same way we style our Mission isocons. 

## Our Mission

_(as it appears in collateral materials)_
The CFPB is focused on making the consumer financial markets work for families by empowering consumers to take more control over their financial lives. 
We are working to foster a marketplace:
- Where customers can see prices and risks upfront and where they can easily make product comparisons. 
- In which no one can build a business model around unfair, deceptive, or abusive practices.
- That works for American consumers, responsible providers, and the economy as a whole.

_(as it appears in the posters at CFPB Headquarters)_
The consumer bureau is a 21st century agency that helps consumer finance markets work 
- by making rules more effective, 
- by consistently and fairly enforcing those rules, and 
- by empowering consumers to take more control of their economic lives. 

_(as it appears on our website)_
Our mission is to make markets for consumer financial products and services work for Americans — whether they are applying for a mortgage, choosing among credit cards, or using any number of other consumer financial products.

_(as it appears in GPRA / Strategic Budget)_
The CFPB is a 21st century agency that helps consumer finance markets work by making rules 
more effective, by consistently and fairly enforcing those rules, and by empowering consumers 
to take more control over their economic lives.

#### Mission isocons (in development)

![cfpb_mission_vision_values_images-01](https://cloud.githubusercontent.com/assets/6887128/4499637/46ea2486-4a85-11e4-9895-99b04aff82e1.png)

#### Educate

An informed consumer is the first line of defense against abusive practices.

![cfpb_mission_vision_values_images-02](https://cloud.githubusercontent.com/assets/6887128/4499642/51c6da16-4a85-11e4-820e-02b7f4000a2f.png)

#### Enforce

We supervise banks, credit unions, and other financial companies, and we enforce federal consumer financial laws.

![cfpb_mission_vision_values_images-03](https://cloud.githubusercontent.com/assets/6887128/4499643/5f5433e0-4a85-11e4-83e9-5451f8d2bdb2.png)

#### Study

We gather and analyze available information to better understand consumers, financial services providers, and consumer financial markets.

How we achieve our Mission and Vision: (This appears in GPRA. I feel that these should get either minicons or no icons at all).

![cfpb_mission_vision_values_images-10](https://cloud.githubusercontent.com/assets/6887128/4499644/67727af0-4a85-11e4-86ff-e9f4c46d8518.png)

##### Data-driven analysis

The CFPB is a data-driven agency. We take in data, manage it, store it, share it appropriately, and protect it from unauthorized access. Our aim is to use data purposefully, to analyze and distill data to enable informed decision-making in all internal and external functions.

![cfpb_mission_vision_values_images-04](https://cloud.githubusercontent.com/assets/6887128/4499649/6c478dd6-4a85-11e4-8b93-3c7669a2ac1d.png)

##### Innovative use of technology

Technology is core to the CFPB accomplishing its mission. This means developing and leveraging technology to enhance the CFPB’s reach, impact, and effectiveness. We strive to be recognized as an innovative, 21st century agency whose approach to technology serves as a model within government.

![cfpb_mission_vision_values_images-07](https://cloud.githubusercontent.com/assets/6887128/4499650/720bf112-4a85-11e4-8b99-cd5d2fe3607c.png)

##### Valuing the best people and great teamwork (not final artwork)

At the CFPB, we believe our people are our greatest asset. Therefore, we invest in world-class training and support in order to create an environment that encourages employees at all levels to tackle complex challenges. We also believe effective teamwork extends outside the walls of the CFPB. We seek input from and collaborate with consumers, industry, government entities, and other external stakeholders. 

### Our Vision

Our vision is a consumer finance marketplace where customers can see prices and risks up front and where they can easily make product comparisons; in which no one can build a business model around unfair, deceptive, or abusive practices; that works for American consumers, responsible providers and the economy as a whole.

### Our Values

We took a look at creating isocons for these but were not entirely successful. If the mission imagery (i.e. missioncons) is isometric, the values should be too. 

#### Serve

![cfpb_mission_vision_values_images-09](https://cloud.githubusercontent.com/assets/6887128/4499658/8d6432bc-4a85-11e4-8cba-12c46a7ef7b2.png)

##### Serve with integrity

Our mission begins with service to the consumer and our country. We serve our colleagues by listening to one another and by sharing our collective knowledge and experience.

#### Lead

![cfpb_mission_vision_values_images-08](https://cloud.githubusercontent.com/assets/6887128/4499662/91a2424c-4a85-11e4-9376-06e3fe5fa676.png)

##### Develop passionate leaders

Fostering leadership and collaboration at all levels is at the core of our success. We believe in investing in the growth of our colleagues and in creating an organization that is accountable to the American people.

#### Innovate

![cfpb_mission_vision_values_images-05](https://cloud.githubusercontent.com/assets/6887128/4499665/96b56a20-4a85-11e4-8237-fb5062313bd1.png)

##### Adapt, improve, and create

Our organization embraces new ideas and technology. We are focused on continuously improving, learning, and pushing ourselves to be great.

**Comment by stephanieosan** [comment]
Is this a duplicate of #253 ? If yes, let's close it.

**Comment by Dnpizarro** [comment]
Its actually not a duplicate. Its addressing the visual inconsistencies of isocon vs minicon-ish approach for our mission and values. But there is some cross over with regard to #253 . This still needs more resolution.

**Comment by nataliafitzgerald** [comment]
In an effort to unpack this issue, I have moved the discussion related to core identity language to a separate issue. https://github.com/cfpb/design-manual/issues/284

So this issue is now only about the specific isocons for "How we will achieve our mission and vision" and "Our Values." 

The specific design needs are: 
- Isometric approach to the Serve, Lead, Innovate (do we still feel like this is a need?). 
- How we will achieve our mission and vision - An alternate approach to the stacked hands. (In our most recent GPRA report we opted for not using imagery for "How we will achieve our mission and vision." I wonder if GPRA is the only use case for this imagery and, if so, perhaps we do not need to address this any further?)

Does that capture it?
